### ISSUE TITLE
Can not get the predicted result

### ISSUE THREAD

**Issue by Bobabc** [wizyoung/YOLOv3_TensorFlow#251]
when i run video_test.py,the code was correctly runned,but there is no predicted result in the window.Can you tell me the reason about that?

**Comment by LyuYifan** [comment]
Maybe the model has not been fully trained?

**Comment by Bobabc** [comment]
emm,i also can not get the predicted result when i used the official weighte file.
And,there is a another question ,the loss in the training process would not decrease after training 30 epochs,the value of training loss is about 15.
how can i finetune the hyperparameters to get a smaller training loss value?
------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "wizyoung/YOLOv3_TensorFlow"                                                                                    <notifications@github.com&gt;;
发送时间:&nbsp;2020年8月1日(星期六) 下午5:09
收件人:&nbsp;"wizyoung/YOLOv3_TensorFlow"<YOLOv3_TensorFlow@noreply.github.com&gt;;
抄送:&nbsp;"1582512089"<1582512089@qq.com&gt;;"Author"<author@noreply.github.com&gt;;
主题:&nbsp;Re: [wizyoung/YOLOv3_TensorFlow] Can not get the predicted result (#251)





 
Maybe the model has not been fully trained?

**Comment by aldinorizaldy** [comment]
The default argument for save_video is False. You should give True to save the video result
